projectatomic/ai-studio
# fix: do not handle pods without ai-studio labels

### What does this PR do?

### What issues does this PR fix or reference?

Fixes #702 

### How to test this PR?

- Create a pod from podman desktop and check that it does not appear in the list of AI Apps
- Create AI Apps and check they appear in the list
